terkin/sensor/system.py

```diff
@@ -135,13 +135,25 @@ def setup(self):
         self.pin = self.settings.get('pin')
         self.resistor_r1 = self.settings.get('resistor_r1')
         self.resistor_r2 = self.settings.get('resistor_r2')
+        self.adc_attenuation_db = self.settings.get('adc_attenuation_db')
```

Comment:
Can we say `self.settings.get('adc_attenuation_db', 6.0)` here in order to make this backward-compatible?